SECURITY.md

```diff
@@ -0,0 +1,98 @@
+# Security Policy
+
+## Supported Versions
+
+The Jaeger project provides community support only for last minor version: bug fixes are released eiter as part of the next minor version or as an on-demand patch version. Independent of which version is next, all patch versions are cumulative, meaning that they represent the state of our `master` branch at the moment of the release. For instance, if the latest version is 1.19.0, bug fixes are released either as part of 1.20.0 or 1.19.1.
+
+Security fixes are given priority and might be enough to cause a new version to be released.
+
+## Reporting a Vulnerability
+
+_The following is a copy of the [Report a security issue](https://www.jaegertracing.io/report-security-issue/) page from our website. The website's version has precedence in case of conflicts._
+
+If you find something suspicious and want to report it, we'd really appreciate!
+
+### Ways to report
+
+* It is preferable to always encrypt your message, no matter the channel you choose to report the issue
+* Contact us on our open chat room on [Gitter][gitter-room]
+* If you can't use Gitter, send a message to [jaeger-tracing@googlegroups.com][mailing-list]
+* If you can't use Gitter nor send an email, open a merge request on GitHub: fork the affected repository and send us a pull request. We really prefer if you'd talk to us before, though, as our repositories are public and we would like to give a heads up to our users before disclosing it publicly
+
+### Our PGP key
+
+No matter what channel you choose to communicate with us, we would prefer you to encrypt your message using our [published key][published-key], available in all major key servers and which should match the one shown below. If you are new to PGP, you can run the following command to encrypt a file called "message.txt":
+
+1. Receive our keys from the key server:
+
+    `gpg --keyserver pool.sks-keyservers.net --recv-keys C043A4D2B3F2AC31`
```

Comment:
Key servers are quite often unavailable. Perhaps we should describe an alternative of simply downloading the public key shown below and importing it from a file. To make that process easier, perhaps it makes sense to have the key in a separate file, so that it can be downloaded directly via URL (unless gpg can ingest this whole file and recognize the key dividers).

Reply:
It's a good practice to have the key in two places, telling users that both should match. In theory, the keyservers have a better security than our website, so, users should be instructed to get it from there.

Let's give this pool a try and see if it's more stable than the instances we've been using.

Another thing we could consider is to use HackerOne for Open Source: https://www.hackerone.com/company/open-source-community